analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -529,7 +529,7 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
             val localRepositories = allRepositories - remoteRepositories
             if (localRepositories.isNotEmpty()) {
                 // No need to use curly-braces-syntax for logging here as the log level check is already done above.
-                logger.debug { "Ignoring local repositories $localRepositories." }
+                logger.debug("Ignoring local repositories $localRepositories.")
```

Comment:
I'd remove the code comment instead because the performance impact is so insignificant. But I'm fine with doing this as well.

Reply:
Since we know that `remoteRepositories` is a subset of `allRepositories` we could do this instead:
```kotlin
if (allRepositories.size > remoteRepositories.size) {
  logger.debug {
    val localRepositories = allRepositories - remoteRepositories
    "Ignoring local repositories $localRepositories."
  }
}
```

Reply:
> I'd remove the code comment instead because the performance impact is so insignificant.

Looks like I'm outvoted here, so I'll change as proposed.